moped-database/migrations/1651095287013_squashed/up.sql

```diff
@@ -0,0 +1,18 @@
+
+CREATE TABLE "public"."moped_user_followed_projects"("id" integer NOT NULL, "created_at" Timestamp NOT NULL DEFAULT now(), "project_id" Integer NOT NULL, "user_id" integer NOT NULL, PRIMARY KEY ("id") , FOREIGN KEY ("user_id") REFERENCES "public"."moped_project"("project_id") ON UPDATE restrict ON DELETE restrict, FOREIGN KEY ("project_id") REFERENCES "public"."moped_users"("user_id") ON UPDATE restrict ON DELETE restrict, UNIQUE ("id"));
+
+alter table "public"."moped_user_followed_projects" drop constraint "moped_user_followed_projects_user_id_fkey";
+
+alter table "public"."moped_user_followed_projects" drop constraint "moped_user_followed_projects_project_id_fkey";
+
+ALTER TABLE "public"."moped_user_followed_projects" ADD COLUMN "id2" serial NOT NULL UNIQUE;
```

Comment:
This statement ends up making a key designation for `id2` along with a `unique` constraint that are orphaned at the end of this migration.
<img width="552" alt="Screen Shot 2022-04-28 at 2 00 37 PM" src="https://user-images.githubusercontent.com/3653206/165827802-01bac5fa-129e-4074-8ad8-ffd3b6d9dfad.png">

Reply:
I believe that reworking the `create_table` migration according to what you suggest below resolves this issue. Can you please confirm?

Reply:
Yep, this one is all squared away. Thanks!